proc_dash/plotting.py

```diff
@@ -64,27 +67,46 @@ def plot_pipeline_status_by_participants(data: pd.DataFrame):
 
 def plot_pipeline_status_by_records(data: pd.DataFrame):
     status_counts = (
-        transform_data_to_long(data)
+        transform_active_data_to_long(data)
         .groupby(["pipeline_name", "pipeline_complete"])
         .size()
         .reset_index(name="records")
     )
 
+    return fig_pipeline_status_by_records(status_counts)
+
+
+def plot_empty_pipeline_status_by_records(pipelines: list, statuses: list):
+    """Generates version of matching records plot for 0 matching records (plot contains no data but correct labels)"""
```

Comment:
I don't find this docstring very clear tbh. If I understand correctly, you are making a dataframe with all possible combinations of pipeline names and completion statuses, and then set the number of records for each of these to 0. The fact that this also makes a plot is not as relevant to understanding the function. And I do think it should not make a plot. Rather `populate_empty_pipeline_completion` as a function makes more sense to me, and then you can feed the output to your plotter wherever you call this function from.